Lib/json/__init__.py

```diff
@@ -342,6 +344,13 @@ def loads(s, *, encoding=None, cls=None, object_hook=None, parse_float=None,
                             f'not {s.__class__.__name__}')
         s = s.decode(detect_encoding(s), 'surrogatepass')
 
+    if encoding is not None:
```

Comment:
Passing `encoding=None` will not emit a warning, but will be an error after removing *encoding* in future Python versions. Use a sentinel.

Reply:
Done, I was unsure if it was worth it.